src/pyOpenMS/pxds/MZTrafoModel.pxd

```diff
@@ -7,17 +7,109 @@ from CalibrationData cimport *
 cdef extern from "<OpenMS/FILTERING/CALIBRATION/MZTrafoModel.h>" namespace "OpenMS":
 
     cdef cppclass MZTrafoModel:
+        # wrap-doc:
+                #   Create and apply models of a mass recalibration function
+                #   -----
+                #   The input is a list of calibration points (ideally spanning a wide m/z range to prevent extrapolation when applying to model)
+                #   -----
+                #   Models (LINEAR, LINEAR_WEIGHTED, QUADRATIC, QUADRATIC_WEIGHTED) can be trained using CalData points (or a subset of them)
+                #   Calibration points can have different retention time points, and a model should be build such that it captures
+                #   the local (in time) decalibration of the instrument, i.e. choose appropriate time windows along RT to calibrate the
+                #   spectra in this RT region
+                #   From the available calibrant data, a model is build. Later, any uncalibrated m/z value can be fed to the model, to obtain
+                #   a calibrated m/z
+                #   -----
+                #   The input domain can either be absolute mass differences in [Th], or relative differences in [ppm]
+                #   The models are build based on this input
+                #   -----
+                #   Outlier detection before model building via the RANSAC algorithm is supported for LINEAR and QUADRATIC models
+
         MZTrafoModel()  nogil except +
         MZTrafoModel(MZTrafoModel &) nogil except +
         MZTrafoModel(bool) nogil except +
-        bool isTrained() nogil except +
-        double getRT() nogil except +
+        bool isTrained() nogil except + # wrap-doc:Returns true if the model have coefficients (i.e. was trained successfully)
+        double getRT() nogil except + # wrap-doc:Get RT associated with the model (training region)
         double predict(double) nogil except +
+            # wrap-doc:
+                #   Apply the model to an uncalibrated m/z value
+                #   -----
+                #   Make sure the model was trained (train()) and is valid (isValidModel()) before calling this function!
+                #   -----
+                #   Applies the function y = intercept + slope*mz + power*mz^2
+                #   and returns y
+                #   -----
+                #   :param mz: The uncalibrated m/z value
+                #   :returns The calibrated m/z value
+
         bool train(CalibrationData, MZTrafoModel_MODELTYPE, bool, double, double) nogil except +
```

Comment:
```suggestion
        bool train(CalibrationData cd, MZTrafoModel_MODELTYPE md, bool use_RANSAC, double rt_left, double rt_right) nogil except +
```

Reply:
If there are multiple parameters which are not named - it is hard to know which ones are the correct ones based on the DocString.